### PR TITLE
Issue 10132 - changed decimals to integers in the arcstat script

### PR DESCRIPTION
**Pull request by Avatat**
### Motivation and Context
The pull request fixes deprecation warning reported here: https://github.com/openzfs/zfs/issues/10132

### Description
I changed time interval and kstat values from decimals to integers, because they are always integers.

### How Has This Been Tested?
I tested it on Ubuntu 20.04 by simple running the script.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
- [ ] I have run the ZFS Test Suite with this change applied.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

**Comment by behlendorf** [comment]
Make sense to me, let's see if we can get @freqlabs or @jwk404 to chime in since they're much more familiar with the right way to handle things in Python.

File: cmd/arcstat/arcstat [hunk 1]
```diff
@@ -335,15 +334,15 @@ def init():
...
 
     if len(argv) > 1:
-        sint = Decimal(argv[0])
+        sint = int(argv[0])
         count = int(argv[1])
 
     elif len(argv) > 0:
-        sint = Decimal(argv[0])
+        sint = int(argv[0])
         count = 0
```
Comment (freqlabs) [inline cmd/arcstat/arcstat]: We should get rid of these lines entirely and fix the previous line as follows:
```python
		     count = int(argv[1]) if len(argv) > 1 else 0
```

Comment (Avatat) [reply cmd/arcstat/arcstat]: Thank you for the suggestion!
The code has been cleaned: https://github.com/openzfs/zfs/pull/10142/commits/eeef30083555efc6cc68501c5e8f4d6617d5ac44
Now it looks much better :)

Comment (Avatat) [reply cmd/arcstat/arcstat]: @freqlabs, it looked fine, but the code changed its behavior.
Before the change:
```
$ arcstat 
    time  read  miss  miss%  dmis  dm%  pmis  pm%  mmis  mm%  arcsz     c  
23:17:56     0     0      0     0    0     0    0     0    0   7.8G  7.7G
```
After the change:
```
$ ./arcstat
    time  read  miss  miss%  dmis  dm%  pmis  pm%  mmis  mm%  arcsz     c  
23:18:00     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:01     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:02     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:03     0     0      0     0    0     0    0     0    0   7.8G  7.7G  
23:18:04     0     0      0     0    0     0    0     0    0   7.8G  7.7G
```

Sorry for the late testing :/ I will prepare a fix for it.

Comment (Avatat) [reply cmd/arcstat/arcstat]: @freqlabs, here is the fix: https://github.com/openzfs/zfs/pull/10142/commits/b51698ffd20aae87807c3ecf11f8bffba138876d
Does it look fine? I added brackets for better readability.

**Comment by freqlabs** [comment]
Thanks, sorry I missed that.

File: cmd/arcstat/arcstat (hunk 1 above)
Comment (freqlabs) [reply cmd/arcstat/arcstat]: Looks good.

**Comment by behlendorf** [comment]
Merged.  @Avatat thanks for fixing this.

**Pull request merged**